model-optimizer/extensions/ops/detectionoutput_onnx.py

```diff
@@ -54,7 +54,7 @@ def infer(node):
         # boxes
```

Comment:
Done for `ExperimentalDetectronDetectionOutput` and `ExperimentalDetectronPriorGridGenerator`. For other operations it will be done in corresponding PRs.